corehq/apps/reminders/util.py

```diff
@@ -65,7 +65,7 @@ def get_form_list(domain):
     form_list = []
     for app_id in get_app_ids_in_domain(domain):
         latest_app = get_app(domain, app_id, latest=True)
-        if latest_app.doc_type == "Application":
+        if latest_app.doc_type in ('Application', 'LinkedApplication'):
```

Comment:
Do remote apps have a specific doc type? Would it it make sense to do something like `latest_app.doc_type != "RemoteApp"`?

Reply:
Have a look at https://github.com/dimagi/commcare-hq/pull/24946, in particular 2e62b082c80b5eecf443d89865df011e331c2937